test/unit/view.child-views.spec.js

```diff
@@ -470,8 +470,8 @@ describe('layoutView', function() {
       this.layoutView.render();
     });
 
-    it('should call empty twice', function() {
-      expect(this.region.empty).to.have.been.calledThrice;
+    it('should call empty once', function() {
```

Comment:
this is a poorly written test, but childviews are no longer emptied on the parent's render